### PR TITLE
Update GoThemis and WasmThemis examples to 0.14.0

### PR DESCRIPTION
**Pull request by ilammy**
Chasing down some stragglers. These ought to be added to the release checklist.

## Checklist

- [x] Change is covered by automated tests
- [x] The [coding guidelines] are followed
- [x] Example projects and code samples are up-to-date
- [x] ~~Changelog is updated~~ (not interesting, IMO)

[coding guidelines]: https://github.com/cossacklabs/themis/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/examples/wasm/webpack/.gitignore [hunk 1]
```diff
@@ -3,4 +3,3 @@
 /dist/main.js
 /dist/main.js.LICENSE.txt
 /node_modules/
-package-lock.json
```
Comment (vixentael) [inline docs/examples/wasm/webpack/.gitignore:6]: hehe

**Review by vixentael — APPROVED** [review]
thank you, makes sense!

**Comment by vixentael** [comment]
> These ought to be added to the release checklist.

o7 will do

**Comment by ilammy** [comment]
I've throw in one more change in README that I missed initially. Will merge without waiting for CI to complete on this branch.

**Pull request merged**